dts/bindings/serial/cdns,uart.yaml

```diff
@@ -0,0 +1,27 @@
+# *************************************************************************
+# Copyright Meta Platforms, Inc. and its affiliates.
+# SPDX-License-Identifier: Apache-2.0
+# *************************************************************************
+
+description: This binding gives a base representation of the CADENCE UART IP6528
+
+compatible: "cdns,uart"
+
+include: [interrupt-controller.yaml, base.yaml]
```

Comment:
is this using `interrupt-controller.yaml`? Also `base.yaml` is not needed, as there is already within `uart-controller.yaml`
Probably just needs `include: uart-controller.yaml`

Reply:
Updated. Removed the interrupt-controller.yaml & base.yaml. Thanks.